ManatiCR/vitaminau
# VIT-60 and VIT-58: Global styles in webforms

# Task [VIT-60](https://manati.atlassian.net/browse/VIT-60)
Define the width of all forms.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the followings urls:
  - http://vitaminau.docker/admin/structure/webform
- Enter each of the forms.
- Check that all forms have a maximum width of 960px and are centered.

# CA
- For example ,it should look like this:
![image](https://user-images.githubusercontent.com/48957125/92757032-9586f280-f34a-11ea-9b21-36bba5e9fdfb.png)

# Task [VIT-58](https://manati.atlassian.net/browse/VIT-58)
Product sale form: multivalued field, ciiu code of the activity, wrap it.


# Test
- Go to the following url: 
  - http://vitaminau.docker/form/sale-of-products.
- Look for the "Código CIIU De La Actividad" field and make sure it looks like this in mobile:
![Screen Shot 2020-09-10 at 7 34 46 AM](https://user-images.githubusercontent.com/48957125/92740745-3a013880-f33b-11ea-927d-d7561512908f.png)
- Verify that the description of the "Sector de actividad económica" field in the beneficiaries section is "Si no se encuentra seguro de la categoría, puede revisar a detalle el Código CIIU https://unstats.un.org/unsd/publication/SeriesM/seriesm_4rev4s.pdf."

# CA:
- It shouldn't look like this in mobile:
![image-20200909-171153](https://user-images.githubusercontent.com/48957125/92741234-a1b78380-f33b-11ea-9e77-42008528dd55.png)
- The "Sector de actividad económica" field must have the following description "Si no se encuentra seguro de la categoría, puede revisar a detalle el Código CIIU https://unstats.un.org/unsd/publication/SeriesM/seriesm_4rev4s.pdf".
